### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by hkershaw-brown**
To review:

Contributors: 
Add Craig Schwartz contributions
other contributors

Check for missing features that should be in there. 

Add date of tag

### CODE REVIEW

**Comment by timhoar** [comment]
Helen ... I cruised through the 202 Fortran files that changed betweeen Manhattan and master.  Here is my summary, how you want to work it into the changelog is up to you.

- removed svn detritus
- cleaner error/log messages (no 'revision','revdate')
- can use character strings to specify inflation algorithm
- Craig Schwartz and Moha fixed something with RTPS ...
- no longer call 'register_module' ... did not provide much info anyway
- preprocess changes
    - new XXX_quantities_mod.f90
    - expanded argument parsing (parse_args_mod.f90)
- expanded support in netcdf_utilities_mod
- matlab routines demonstrating inflation now mirror fortran implementation
- Adding E_CONTINUE to allow programs to carry on after throwing an error
    - allows for more thorough debugging.
- obs_sequence_tool Chris Riedel ... write_external_FOs -> remove_precomputed_FO_values
      - tool now writes out all precomputed values unless explicitly specified. Previous versions simply removed all precomputed FOs all the time ...
- rttov_unit_tests.f90  ... new ... uses TERMLEVEL 3
- mpas_atm & rttov support .. QTY_CLOUD_FRACTION,
     - improved interpolation for 2D fields not at surface ... (avoids 'sfc_elev_max_diff' check)
- helen ... you made changes to support precomputed FO copies in an MPI environment ...
- support for more Flux Tower observations  (obs_def_tower_mod.f90)
- AIRS/airs_JPL_mod.f90 strictly supports HDF-EOS2 (not HDF-EOS5) and is only used for Level 2 (i.e. retrievals of) temperature and humidity observations.
- new AIRS/convert_amsu_L1.f90, amsua_bt_mod.f90 to support converting AMSUA brightness temperatures to obs_seq

For later ... 
** guide/boilerplate/html_*.f90 ... delete?

**Comment by hkershaw-brown** [comment]
cool thanks Tim! I'll get them added.

**Comment by hkershaw-brown** [comment]
> For later ...
> ** guide/boilerplate/html_*.f90 ... delete?
oh yeah, massive purge.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -16,12 +16,74 @@ output of ``git log``
...
 individual files.
 
 The changes are now listed with the most recent at the top.
 
+**April 26 2021 :: preprocess, inflation options, external FO output, 
+wrf-hydro, AMSU-A, DART_LAB Tag: v9.10.0**
+
+*New features*
+
+  -  Updated ``preprocess``:
```
Comment (nancycollins) [inline CHANGELOG.rst:30]: these sections (items starting with dashes) are indented one level which makes it look strange when formatted.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -16,12 +16,74 @@ output of ``git log``
 
    0[1011] machine:dartGIT % git log > full_git_log.txt
```
Comment (nancycollins) [inline CHANGELOG.rst:17]:
```suggestion
   % git log > full_git_log.txt
```
the complex prompt stuff is slightly confusing.

**Comment by nancycollins** [comment]
i'm fine with telling people to check out a tag after they've cloned the repo, but i think this info also needs to be someplace more prominent than just in the changelog.  the readme.md needs to say what tag to check out - maybe at the start of the 'quickstart for the impatient' section?

File: CHANGELOG.rst (hunk 1 above)
Comment (timhoar) [reply CHANGELOG.rst:30]: I agree with Nancy, the formatting looks odd ... must fix before merging.

**Review by timhoar — CHANGES_REQUESTED** [review]
The changelog formatting is not right ... maybe the colon after ``preprocess`` ? ... ditto after MPAS?

File: CHANGELOG.rst (hunk 2 above)
Comment (timhoar) [reply CHANGELOG.rst:17]: Says you :)!  Having just a percent to indicate that this is a unix prompt is a bit spartan for my taste. People might just copy the thing in its entirety.

File: CHANGELOG.rst (hunk 1 above)
Comment (hkershaw-brown) [reply CHANGELOG.rst]: I believe (?) I fixed the formatting. will review in person.

**Comment by nancycollins** [comment]
this branch seems to have a readme.md AND a readme.rst file, which are completely different.  which one is going to be "the readme" once the pull request is committed?  the readme.md has (correct) terse download instructions including the tag info, but the readme.rst is much longer, has more sections, and has other info in the quickstart.

**Comment by hkershaw-brown** [comment]
> this branch seems to have a readme.md AND a readme.rst file, which are completely different. which one is going to be "the readme" once the pull request is committed? the readme.md has (correct) terse download instructions including the tag info, but the readme.rst is much longer, has more sections, and has other info in the quickstart.

The README.md is to have a short readme on Github.   The README.rst displays the Sphinx directives (e.g. the ..table of contents) on github. 

README.rst is the index page for readthedocs. It may make more sense to rename README.rst to index.rst so we don't have two readmes.

File: CHANGELOG.rst (hunk 1 above)
Comment (timhoar) [reply CHANGELOG.rst]: yup ... looks good now.

File: README.md [hunk 3]
```diff
@@ -6,17 +6,31 @@
...
+
+#### Downloading DART
+
+DART is available through GitHub. To download the latest version of DART:
+
+```
+git clone https://github.com/NCAR/DART.git
+cd DART
+git checkout tags/v9.10.0 -b v9.10.0-branch
+```
```
Comment (timhoar) [inline README.md:26]:
```suggestion
git checkout tags/v9.10.0 -b v9.10.0-branch
```
git checkout -b v9.10.0-branch tags/v9.10.0
```

File: README.rst [hunk 4]
```diff
@@ -80,13 +80,21 @@ Quick-start
...
 1. fork the NCAR/DART repo
 2. clone your (new) fork to your machine - this will set up a remote named
-   ‘origin’
+   ‘origin’. To clone DART and checkout the latest release, use
+
+.. code::
+
+   git clone https://github.com/USERNAME/DART.git
+   cd DART
+   git checkout tags/v9.10.0 -b v9.10.0-branch
```
Comment (timhoar) [inline README.rst:89]:
```suggestion
   git checkout -b v9.10.0-branch tags/v9.10.0 
```

File: guide/downloading-dart.rst [hunk 5]
```diff
@@ -12,13 +12,25 @@ for more information on how to clone a repository. Take note of the
 directory you installed into, which is referred to as ``DART`` throughout 
 this documentation.
 
+To checkout the latest release of DART:
+
+.. code:: 
+
+   git clone https://github.com/NCAR/DART.git
+   cd DART
+   git checkout tags/v9.10.0 -b v9.10.0-branch
```
Comment (timhoar) [inline guide/downloading-dart.rst:21]:
```suggestion
   git checkout -b v9.10.0-branch tags/v9.10.0 
```

**Review by timhoar — APPROVED** [review]
All looks good.

File: README.md [hunk 6]
```diff
@@ -6,17 +6,31 @@
...
+To subscribe to the list, click on [Dart-users](http://mailman.ucar.edu/mailman/listinfo/dart-users). 
+
+#### Downloading DART
+
+DART is available through GitHub. To download the latest version of DART:
+
+```
+git clone https://github.com/NCAR/DART.git
+cd DART
+git checkout tags/v9.10.0 -b v9.10.0-branch
```
Comment (timhoar) [inline README.md:25]:
```suggestion
git checkout -b v9.10.0-branch tags/v9.10.0 
```

File: guide/downloading-dart.rst (hunk 5 above)
Comment (timhoar) [inline guide/downloading-dart.rst:21]:
```suggestion
   git checkout -b v9.10.0-branch tags/v9.10.0 
```

**Comment by nancycollins** [comment]
i'd vote for index.rst and README.md for sanity's sake.

File: CHANGELOG.rst [hunk 7]
```diff
@@ -14,14 +14,86 @@ output of ``git log``
 
 ::
 
-   0[1011] machine:dartGIT % git log > full_git_log.txt
+   0[1011] machine:DART % git log > full_git_log.txt
```
Comment (nancycollins) [inline CHANGELOG.rst:17]: ok how about forget the prompt completely and make it: 
```suggestion
git log > full_git_log.txt
```

**Review by nancycollins — APPROVED** [review]
other than 1) a second suggestion to simplify the example git command even further, and 2) supporting the suggestion to rename one of the READMEs to avoid confusion, the rest of this looks good now.

**Pull request merged**